### ISSUE TITLE
Only count time of activity

### ISSUE THREAD

**Issue by arty-name** [slgobinath/SafeEyes#26]
It happens quite often that I sign in into my computer only to be greeted with „take a walk“ or similar screen after maybe 15 seconds of using it. I guess the app would benefit from ignoring the periods of time which OS considers inactive.

**Comment by Semmu** [comment]
How exactly would you determine inactivity? A trivial method would be looking for screen lock/unlock events, but is there more? Mouse movement tracking could also be used, but it may be too much (for privacy reasons).

**Comment by slgobinath** [comment]
Hi @Semmu,
Right now I am working on this issue. The `xprintidle` tool provides the idle time using system records. I am using this tool to get the idle time and based on that, the Safe Eyes will pause/resume itself.

**Comment by arty-name** [comment]
@Semmu I assumed that such a common task is already solved in various desktop environments via events or something

**Comment by slgobinath** [comment]
Hi,

This issue has been resolved and released in Safe Eyes [v1.1.4](https://github.com/slgobinath/SafeEyes/releases/tag/v1.1.4). This how it works:
In Settings, you can specify the minimum idle time to pause Safe Eyes. The default value is 5 minutes.
If the user does not interact with the computer at least for 5 minutes, Safe Eyes will pause itself.
Once the user starts using the computer again, Safe Eyes will resume and schedule the next break from that time.
In other words, if the user does not use the computer for 5 minutes, Safe Eyes assumes that user does not need a break in next 15 minutes.
